test/cmd/get.sh

```diff
@@ -353,7 +353,7 @@ run_kubectl_all_namespace_tests() {
   # Pre-condition: node exists
   kube::test::get_object_assert nodes "{{range.items}}{{$id_field}}:{{end}}" '127.0.0.1:'
   # Command
-  output_message=$(kubectl get nodes --all-namespaces 2>&1)
+  output_message=$(kubectl get nodes -A 2>&1)
```

Comment:
this might not be the best test of the -A flag, since the point is for it to not print namespace for cluster-scoped resources

would adding to the earlier call that does `kubectl get serviceaccounts --all-namespaces` to also add a call that uses `-A` be better?

Reply:
Yeah, I was considering adding an identical test to an existing one in other place. I'll update that.